lib/module.js

```diff
@@ -71,22 +71,117 @@ module.exports = async function (moduleOptions) {
         "sectionsNotLoadedCorrectly": "Some sections could not be loaded correctly, saving the page will delete these sections from your page, unless you are happy with the page you see now, do not save it",
         "fieldNames": "Image/Media fields",
         "fieldDesc": "Declare the name of the fields carrying images/medias in your section, so that Sections can properly process them and return the URL for your files",
+        "pathFieldDesc": "The path of the page cannot be blank, must be unique and does not accept specials characters",
         "field": "Field",
         "exportSectionsLabel": "Export sections",
         "importSectionsLabel": "Import sections",
         "clickToCopy": "click to copy",
         "deletePage": "Delete page",
         "delete-section-page": "Are you sure you want to delete this page ?",
         "settingsSectionsLabel": "Page settings",
-        "pageTitle": "Title in english",
-        "pageTitleFr": "Title in french",
-        "pageSeoDesc": "SEO description in english",
-        "pageSeoDescFr": "SEO description in french",
+        "pageTitle": "Title",
+        "pageSeoDesc": "SEO description",
         "pageUrl": "Page path*",
         "savePageSettings": "To save the metadata and the page url, you need to save your page",
         "imageFieldValidation": "image field must be an array for section ",
         "pagePathRequired": "Page path is required",
-        "Metadata": "Settings"
+        "Metadata": "Settings",
+        "mediaComponent": {
```

Comment:
They can, but I actually couldn't find a way to merge these translation into the host project translation where the vue component is installed at. So I had to relay them to the module translations.

This is because inside module.js I don't have access to media meta component translations of the vue-components library